Lib/asyncio/streams.py

```diff
@@ -610,6 +626,9 @@ class StreamReaderProtocol(FlowControlMixin, protocols.Protocol):
 
     def __init__(self, stream_reader, client_connected_cb=None, loop=None,
                  *, _asyncio_internal=False):
+        warnings.warn("The loop argument is deprecated since Python 3.8, "
```

Comment:
This class is actually *private API*. Please remove the warning, I pretty sure that asyncio factories like `open_connection` always send the `loop`. Double deprecation is redundant.

Reply:
If so it should  be documented as private, it does not appear to be : https://docs.python.org/3.6/library/asyncio-stream.html#streamreaderprotocol

And a quick search returns case of people passing the loop argument to it. 
https://github.com/ssfdust/pyclewn/blob/555f2e5485f9938078adfe19388a0e1bb8af67f5/lib/clewn/tty.py#L119

Reply:
Agree, it should be documented properly.
I've added `_asyncio_internal` flag.
Also, `asincio.__init__` has `__getattr__()` now to raise `DeprecationWarning` for importing `StreamReader`, `StreamWriter` and `StreamReaderProtocol`.
We have an agreement with @1st1 to land asyncio streams code changes first and document them later before Release Candidate stage.

I supposed to make a PR for documentation. If somebody would help me -- I very appreciate this.
Especially taking into account that my English is rough :(
Usually Yuri replaces almost every my sentence on review :)

Reply:
Hi @asvetlov I will help with the docs. :-)